docs/getting-started/releases.md

```diff
@@ -9,80 +9,67 @@ order: 8
-> - Set up a `.env` file with your Github and npm account information (see `.env.example`).
-A. __To update the stable branch for a patch release__ (e.g. 1.4.1 when the `master` branch is already at 2.0.0):
-
-1. Run `git fetch --all --tags`.
-1. Run `git checkout -B your_local_branch --track origin/stable`.
-1. Merge in the latest release tag into the stable branch: `git merge v1.4.1`
-1. Update the remote stable branch: `git push origin stable`
-
-B. __To add a change to the patch release:__
-
-1. Run `git fetch --all --tags`.
-1. Run `git checkout -B your_local_branch --track origin/stable`.
-1. Make the bug fix and commit the changes.
-1. Push the change to gerrit for review: `git push origin HEAD:refs/for/stable`
-1. Review the new commit, test it and merge it to the remote stable branch.
-
-C. __To create the patch release commit:__
-
-1. Run `git fetch --all --tags`.
-1. Run `git checkout -B your_local_branch --track origin/stable`.
-1. Run `yarn bump`.
-1. Check that the CHANGELOG.md files and version numbers were updated correctly.
-1. Push the version bump commit for review: `git push origin HEAD:refs/for/stable`
-1. Review the version bump commit, test it and merge it.
-
-D. __To manually run the patch release:__
-
-> __Before beginning you will need:__
-> - Permissions to publish to the instructure org on npm.
-> - Set up a `.env` file with your Gerrit and npm account information (see `.env.example`).
-1. Run `git fetch --all --tags`.
-1. Run `git checkout -B your_local_branch --track origin/stable`.
-1. Run `yarn release patch`. (Note the tag ('patch') option here--this is important if the latest release is already ahead of the patch release. If not, you can just run `yarn release` or let the CI build handle it.)
 1. Verify that the release was [tagged in Github](https://github.com/instructure/instructure-ui/releases).
-1. Verify that the release was published to Npm by running `yarn info [package]@[version]`.
 
-E. __Merge the release into the master branch:__
+1. Verify that the release was published to npm by running `yarn info [package]@[version]`.
+
+1. Verify that the [documentation was updated](https://instructure.design/).
+
+### Patching Older Releases
 
-1. Run `git fetch --all --tags`.
-1. Run `git checkout -B your_local_branch --track origin/master`.
-1. Run `git merge origin/stable`
-1. Push the merge commit for review: `git push origin HEAD:refs/for/master`
-1. Review the merge commit, test it and merge it.
+0. You need to branch off from the **last release commit** of the old version and name it v< major-version> -maintenance, if it is non existent. E.g. if you'd like to start v6 maintenance branch:
+
+```
+//checkout the v6.26.0 commit
+git checkout 35258f25cad1e17e5dbe5dcb4389b5185d709f72
+
+//create new branch from it
+git checkout -b v6-maintenance
+```
+
+1.  Check out the v< major-version >-maintenance branch. E.g. for v7:
+
+```
+git checkout v7-maintenance
```

Comment:
It is a bit confusing for me that first you use v6 as an example, then v7 in the next. Which one do I need to checkout now? Both?

Reply:
Fixed wording